db/migrate/20170116143350_add_name_to_tables.rb

```diff
@@ -0,0 +1,5 @@
+class AddNameToTables < ActiveRecord::Migration
+  def change
+    add_column :tables, :name, :string, default: ''
```

Comment:
Since we're allowing search in this column, we must add index to the column:
```ruby
add_column :tables, :name, :string, default: '', index: true
```